include/poketch_data.h

```diff
@@ -1,51 +1,98 @@
 #ifndef POKEPLATINUM_POKETCH_DATA_H
 #define POKEPLATINUM_POKETCH_DATA_H
 
-#include "struct_decls/struct_poketch_data_decl.h"
 #include "pokemon.h"
 #include "struct_decls/struct_021C0794_decl.h"
 
+/* Poketch app IDs, used to register and switch between apps in the Poketch */
+typedef enum PoketchAppID {
```

Comment:
Convention in the project is to actually *not* typedef enums, to clearly delineate them from structs.